cypress/integration/login.js

```diff
@@ -20,5 +20,6 @@ describe('Login functionality', function() {
         cy.get('.tile.form').should('be.visible')
         cy.get('.user-info__title').contains('Demo User')
         cy.get('.main-nav').find('li').should('have.length', 1)
+        cy.get('.calendar--selected').contains(15)
```

Comment:
1. This will fail if you'll select a different day in the calendar. (the locator is incorrect)
2. Additionally, this test will fail as tomorrow will be the 16th, e.g. maybe we can always know the todays day? e.g. a hint https://www.w3schools.com/jsref/jsref_getday.asp

Reply:
https://www.w3schools.com/jsref/jsref_getdate.asp